openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -0,0 +1,69 @@
+"""
+Tagging API Views
+"""
+from rest_framework import status
+from rest_framework.permissions import DjangoModelPermissions
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from ...api import (
+    create_taxonomy,
+    delete_taxonomy,
+    get_taxonomy,
+    get_taxonomies,
+    update_taxonomy,
+)
+from .serializers import TaxonomyListQueryParamsSerializer, TaxonomySerializer
+
+
+class TaxonomyView(ModelViewSet):
+    """
+    Docstring
+    """
+
+    serializer_class = TaxonomySerializer
+    lookup_field = "id"
+    permission_classes = [DjangoModelPermissions]
```

Comment:
If you don't need any custom permissions, then you don't need a `permissions.py` file either.

But if you do end up needing custom permissions, I recommend the approach [described here](https://www.django-rest-framework.org/api-guide/permissions/#djangomodelpermissions):

> To use custom model permissions, override `DjangoModelPermissions` and set the `.perms_map` property. Refer to the source code for details.

This post describes it well: https://nezhar.com/blog/django-rest-framework-permissions-in-depth/